docs/DeveloperGuide.md

```diff
@@ -476,6 +529,9 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
     * 2d1. ModuleBook 3.5 shows an error message.
 
       Use case resumes at step 2.
+    
+    
+Use case 9:
```

Comment:
Not sure why there is a new Use case 9 right before the NFR section.

Reply:
resolved :)